man/systemd.resource-control.xml

```diff
@@ -710,6 +710,42 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SocketBindAllow=<replaceable>BIND_RULE</replaceable></varname></term>
+        <term><varname>SocketBindDeny=<replaceable>BIND_RULE</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow or deny binding a socket address to a socket by matching it with the <replaceable>BIND_RULE</replaceable> and
+          applying a corresponding action if there is a match.</para>
+
+          <para><replaceable>BIND_RULE</replaceable> describes socket properties such as <replaceable>ADDRESS_FAMILY</replaceable>
+          and <replaceable>IP_PORTS</replaceable>.</para>
+
+          <para><replaceable>BIND_RULE</replaceable> := [<replaceable>ADDRESS_FAMILY</replaceable><constant>:</constant>]<replaceable>IP_PORTS</replaceable></para>
+
+          <para><replaceable>ADDRESS_FAMILY</replaceable> := { <constant>IPv4</constant> | <constant>IPv6</constant> }</para>
+
+          <para><replaceable>IP_PORTS</replaceable> := { <replaceable>IP_PORT</replaceable> | <replaceable>IP_PORT_RANGE</replaceable> |
+          <constant>any</constant> }</para>
+
+          <para>An optional <replaceable>ADDRESS_FAMILY</replaceable> expects <constant>IPv4</constant> or <constant>IPv6</constant> values.
+          If not specified, a rule will be matched futher for both IPv4 and IPv6 addresses and applied depending on other socket fields, e.g. user_port.</para>
+
+          <para><replaceable>IP_PORT</replaceable> value must lie within <constant>[1, 65535]</constant> interval
```

Comment:
Square brackets already mean "inclusive". We standardized on using "…" for ranges. So I think this should say "within 1…65535 inclusively".